lib/routes/password.js

```diff
@@ -149,7 +149,7 @@ module.exports = function (
         var wantsKeys = requestHelper.wantsKeys(request)
         var account, verifyHash, sessionToken, keyFetchToken
 
-        return changePassword()
+        changePassword()
```

Comment:
I don't believe we need it, https://github.com/mozilla/fxa-auth-server/blob/edecb24ec75b6193bf2bda0ecdb2476801a5a427/lib/routes/password.js#L49.